src/windows/find_tools.rs

```diff
@@ -493,18 +496,18 @@ mod impl_ {
                     return None;
                 }
                 let mut tool = Tool::with_family(path, MSVC_FAMILY);
-                if target == "x86_64" {
+                if target == TargetArch::X86 {
```

Comment:
Given that it originally matches against `x86_64`, I think it should be `X64`

```suggestion
                if target == TargetArch::X64 {
```

Reply:
Whoops, nice catch!